sdk/resourcemanager/Azure.ResourceManager/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.0.0-beta.9 (Unreleased)
```

Comment:
Hold on for a while as there is a user agent issue

Reply:
It's already realeased.